src/BaseApiAbstract.php

```diff
@@ -299,7 +299,8 @@ private function processBodyOptions(array $requestData = [])
             if (is_bool($value)) {
                 $value = (int)$value;
             }
-            if ('file' === $key) {
+            // "File" key is used in FIleAPI and "Content" in ContextAPI.
+            if (in_array($key, ['file', 'content'])) {
```

Comment:
Maybe we just need to override this method in successors? The the base method just returns
```
return $requestData;
```
File of Context classes will override value of the single key